mails/themes/modern_mjml/core/productoutofstock.mjml.twig

```diff
@@ -37,7 +37,7 @@
 <mj-section padding="15px 50px 40px">
   <mj-column background-color="#fefefe" border="1px solid #DFDFDF" padding-top="10px" padding-bottom="10px">
     <mj-text>
-      {{ 'There are now less than [1]{last_qty}[/1] items in stock.'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
+      {{ 'There are now less than [1]{last_qty}[/1] of [1]{product}[/1] in stock.'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
```

Comment:
```suggestion
      {{ 'There are now less than [1]{last_qty}[/1] items of [1]{product}[/1] in stock.'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
```

Reply:
I'm fine with it but it will add yet another wording in the catalog. `There are now less than [1]{last_qty}[/1] of [1]{product}[/1] in stock.` is present in the 177 catalog for now. Should we add this new wording for the patch version then?

Reply:
@Julievrz what do you think?

Reply:
Thanks 👍